src/sentry/api/endpoints/organization_events_facets_performance.py

```diff
@@ -30,6 +35,19 @@ def get(self, request, organization):
         filter_query = request.GET.get("query")
         aggregate_column = request.GET.get("aggregateColumn")
 
+        all_tag_keys = None
+        tag_key = None
+        histograms = None
+
+        if self.has_tag_page_feature(organization, request):
+            # TODO(k-fish): Temporarily allow these options for this feature, potentially circle back to split histogram off before release.
```

Comment:
by this do you mean split histograms to its own endpoint?

Reply:
Not sure, thoughts? I personally prefer separate endpoints for different data structures. What did you have in mind?

Reply:
yeah IMO we shouldn't change the response structure of this endpoint based on params. so either a param for the existing histogram endpoint or a completely separate one

Reply:
A separate endpoint would be amazing.